src/clr-angular/utils/drag-and-drop/draggable.spec.ts

```diff
@@ -41,24 +92,168 @@ export default function(): void {
                 expect(this.dragHandleRegistrar.customHandleEl).toBeUndefined();
             });
 
+            it("should instantiate cloned version of draggable as ghost on drag start", function() {
```

Comment:
Aren't you also testing that the ghost is cloned right next to the draggable? We should update the description accordingly.

Reply:
Addressed this.